docker_build_and_push.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#building the docker file
+if [ $# -eq 0 ]; then
+  echo "starting the docker building of material-api:latest as no argument provided."
+  docker build -t rebirthbridge/material-api .
+  echo "docker push will not happen since there is no argument for versioning. \
+  Since there is no argument provided, script has built the latest, which will not be pushed to remote. \
+  Starting the container assuming you just want to start locally.."
+  docker run -p 9000:3010 -d rebirthbridge/material-api:latest
```

Comment:
Should this be `9000:8080` ?

Reply:
Changed the app starting point to 8080. docker has the upstream to 8080 in scripts now. local port I kept the 3010 to avoid developer confusion. We can change this later.